internal/providers/amazon/eks/workflow/bootstrap_activity.go

```diff
@@ -153,6 +156,32 @@ func (a *BootstrapActivity) Execute(ctx context.Context, input BootstrapActivity
 		}
 	}
 
+	ds, err := kubeClient.AppsV1().DaemonSets("kube-system").Get("aws-node", metav1.GetOptions{})
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get CNI driver daemonset")
+	}
+
+	tags := map[string]string{}
+
+	for _, tag := range amazon.PipelineTags() {
+		tags[aws.StringValue(tag.Key)] = aws.StringValue(tag.Value)
+	}
+
+	tagBody, err := json.Marshal(tags)
+	if err != nil {
+		return nil, cadence.NewClientError(err)
+	}
+
+	ds.Spec.Template.Spec.Containers[0].Env = append(ds.Spec.Template.Spec.Containers[0].Env, v1.EnvVar{
```

Comment:
I think we first should check if `ADDITIONAL_ENI_TAGS` is already set or not. If `ADDITIONAL_ENI_TAGS` already exists we need to merge the Pipeline tags with the existing values under `ADDITIONAL_ENI_TAGS`.

Reply:
The default value is `{}`. In case of a new cluster I doubt this will have any initial values.